app/controllers/concerns/vendor_api/application_data_concerns.rb

```diff
@@ -0,0 +1,45 @@
+module VendorAPI
+  module ApplicationDataConcerns
+  private
+
+    def application_choice_id
+      params.permit(:application_id)[:application_id]
+    end
+
+    def application_choice
+      @application_choice ||= application_choices_visible_to_provider.find(application_choice_id)
+    end
+
+    def render_application
+      render json: application_json
+    end
+
+    def application_json
+      data = { data: ApplicationPresenter.new(version_number, application_choice).as_json }
+      data.merge!(meta) if version_number > '1.0'
```

Comment:
👍  for this not being in the cache as would cause issues esp with timestamp, however is there a way we could add it in the change modules 🤔

Reply:
💡

Reply:
Added a ticket so we can tackle this seperately